flow/core/vehicles.py

```diff
@@ -24,7 +24,6 @@
 
 class Vehicles:
     """Base vehicle class.
-
```

Comment:
all the deleted spaces in this file should not have been deleted as per documentation standards

Reply:
Ty, will fix.

Reply:
@AboudyKreidieh we should formalize these standards in a customized style checker.

Reply:
yup, this is part of #189 , but it's been low priority for a while